doc-content/enterprise-only/integration/entando-con.adoc

```diff
@@ -2,6 +2,8 @@
 
 = Digital Experience Platform (DXP) and AppBuilder
 
+WARNING: {PRODUCT} will no longer include an entitlements for the Entando AppBuilder in 7.8.0 release. The support for Entando AppBuilder is deprecated and it will be removed in a future 7.9.0 release.
```

Comment:
@kaldesai the use of the word "entitlements" gives me pause. Would "privileges" be more appropriate?

Reply:
How about: 

WARNING: {PRODUCT}{ENTERPRISE_VERSION} no longer includes Entando AppBuilder privileges. Support for Entando AppBuilder is deprecated and will be removed in {PRODUCT} 7.9.0.

Reply:
Or, 

WARNING: {PRODUCT}{ENTERPRISE_VERSION} no longer includes access to Entando AppBuilder. Support for Entando AppBuilder is deprecated and will be removed in {PRODUCT} 7.9.0.

Reply:
I agree with makes the statement present tense. However, perhaps 'entitlements' is correct. Maybe check with the SME - it could be a legal designation. Also, I think we avoid say specifically which release it will be removed in in case it doesn't happen. So .. will be removed in a future release.

Also. I like the original {PRODUCT} no longer includes...
If you include the release number,  you are implying that at some point 7.8 did include it but now doesn't. So perhaps:

Starting with release {ENTERPRISE_VERSION}, {PRODUCT} no longer includes entitlements for Entando AppBuilder. Support for Entando AppBuilder is deprecated and will be removed in a future release.

Reply:
@michelehaglund and @emmurphy1, thank you for reviewing the note. I confirmed with the SME regarding the `entitlements` word usage and they are okay with it. Apart from that, I will incorporate your suggested feedback.